<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_input_file" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_zip_file">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Zip File:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_zip_file">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>431</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_unzip_path">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>431</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_unzip_dir">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>UnZip Dir:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_choose_file">
     <property name="geometry">
      <rect>
       <x>527</x>
       <y>17</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>choose file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_choose_dir">
     <property name="geometry">
      <rect>
       <x>527</x>
       <y>57</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>choose dir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_find_message">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Message</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_display_file_list">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>110</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Display File List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_export_result">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Export Result</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_rule" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>641</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_rule_and">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>10</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rule_or">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>10</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>OR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rule_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rule Type:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_rule_display">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>33</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_rule_item1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>36</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rule Item1:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rule_item1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>36</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_rule_item2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rule Item2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rule_item2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>66</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_rule_item3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>96</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rule Item3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rule_item3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>96</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rule_add">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>128</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rule_delete">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>128</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rule_clear">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>128</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rule_step">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>STEP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_match_action">
     <property name="geometry">
      <rect>
       <x>348</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Match Action: Display</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_display_rows">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_match_display_rows">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rows</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="result_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1271</width>
      <height>651</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1269</width>
       <height>649</height>
      </rect>
     </property>
     <widget class="QPlainTextEdit" name="display_result_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1271</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
